### PR TITLE
Add model verbosity attribute

### PR DESCRIPTION
**Pull request by fonnesbeck**
Adds a simple flag for turning on and off feedback, such as that regarding auto-assignment of step methods.

### CODE REVIEW

File: pymc3/model.py [hunk 1]
```diff
@@ -158,9 +158,15 @@ def logpt(self):
...
+    """Encapsulates the variables and likelihood factors of a model.
+    
+    Parameters
+    ----------
+    verbose : int
+        Setting for model feedback verbosity (Defaults to 1 -- normal verbosity).
+    """
 
-    def __init__(self):
+    def __init__(self, verbose=1):
```
Comment (twiecki) [inline pymc3/model.py:169]: why not `True`?

Comment (fonnesbeck) [inline pymc3/model.py:169]: At the moment it is evaluated as if `True`, but we may eventually want to have 2-3 verbosity levels, perhaps for debugging.

**Comment by fonnesbeck** [comment]
This is intended to address this sort of thing:

**Comment by fonnesbeck** [comment]
OK, I have no idea why this is failing.

**Comment by fonnesbeck** [comment]
Giving up on this.

**Pull request closed without merging**